### DISCUSSION TITLE
Save Applied Sorting value if refresh the page?

### DISCUSSION THREAD

**Discussion by gal01azabolcs** [rappasoft/laravel-livewire-tables#581] [Q&A]
It is there any solution to get applied sorting colums, after refresh the page?

**Comment by rappasoft** [comment]
The current sorts are saved in the query string, if for some reason you want to do something with them in the component, you can access them with `$this->sorts`.

**Reply by gal01azabolcs** [reply]
@rappasoft can you show me a simple example with this,  if it's not too much to ask? :D

**Reply by rappasoft** [reply]
I don't understand what you need an example of?

**Reply by gal01azabolcs** [reply]
I want to keep the filters and sorts even after the page is refreshed by the user...and I don't knwo if it is possible?

**Reply by rappasoft** [reply]
That happens by default. If you go [to this example](https://tables.laravel-boilerplate.com/tailwind?sorts[name]=asc&sorts[email]=asc&filters[search]=kozey&filters[verified]=yes) everything should be preselected.

If you're talking about saving the state in the session so if they closed the browser and came back and everything was still there, then no that is not supported.

**Reply by gal01azabolcs** [reply]
Ohh sorry I just noticed that this is really the case, I've only tested it on the ngnix server and doesn't work there :(
